microsoft/PowerToys
# [KBM] Removed re-reading loop in settings view model (#18098)

## Summary of the Pull Request

**What is this about:**
 `KeyboardManagerViewModel.LoadProfile` expected the mapping settings file to be created by KBM process.
In absence of the file several attempts to read the settings file was done.

**What is included in the PR:** 
Removed re-reading attempts. If no settings file exists we will create it on save.

**How does someone test / validate:** 
1. Install Power Toys
2. Erase `%LOCALAPPDATA%\Microsoft\PowerToys\Keyboard Manager` folder
3. Open Power Toys Settings
4. Switch to Keyboard Manager
5. UI should not freeze

## Quality Checklist

- [x] **Linked issue:** #18098

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
